server_addon/create_ayon_addons.py

```diff
@@ -191,22 +205,37 @@ def create_addon_package(
     addon_output_dir.mkdir(parents=True)
 
     # Copy server content
-    src_root = os.path.normpath(str(server_dir.absolute()))
-    src_root_offset = len(src_root) + 1
-    for root, _, filenames in os.walk(str(server_dir)):
-        dst_root = addon_output_dir
-        if root != src_root:
-            rel_root = root[src_root_offset:]
-            dst_root = dst_root / rel_root
-
-        dst_root.mkdir(parents=True, exist_ok=True)
-        for filename in filenames:
-            src_path = os.path.join(root, filename)
-            shutil.copy(src_path, str(dst_root))
+    if not rez_package:
+        src_root = os.path.normpath(str(server_dir.absolute()))
+        src_root_offset = len(src_root) + 1
+        for root, _, filenames in os.walk(str(server_dir)):
+            dst_root = addon_output_dir
+            if root != src_root:
+                rel_root = root[src_root_offset:]
+                dst_root = dst_root / rel_root
+
+            dst_root.mkdir(parents=True, exist_ok=True)
+            for filename in filenames:
+                src_path = os.path.join(root, filename)
+                shutil.copy(src_path, str(dst_root))
+    else:
+        package_py = addon_output_dir / "package.py"
+
+        with open(package_py, "w+") as pkg_py:
+            pkg_py.write(
+                f"""name = "{addon_dir.name}"
+version = "{addon_version}"
+plugin_for = ["ayon_server"]
+build_command = "python {{root}}/rezbuild.py"
```

Comment:
What is this for?

Reply:
It's for when running the installation of the package on the server side, it will run the [`rezbuild.py` in the backend](https://github.com/ynput/ayon-backend/pull/29/files#diff-3971b40e9cd7b57b0924840af2ddb6af6c487eb73376805823ffe2ec6e517666)

Which as far as I recall, would be the default for simple add-ons.

Initially tried to add this field on the fly if it was missing, but there's a but in rez which won't allow to do such thing.

Reply:
In that case I don't think it is necessary
```suggestion
```